### ISSUE TITLE
Support crl-0.27.0 undefined epoch_ms

### ISSUE THREAD

**Issue by duraki** [robacarp/mosquito#23]
When we try to enqueue a job to task store, we get an issue:

```
in lib/mosquito/src/mosquito/queued_job.cr:108: instantiating 'Mosquito::Task#store()'

      task.store
           ^~~~~

in lib/mosquito/src/mosquito/task.cr:46: undefined method 'epoch_ms' for Time

      epoch = time.epoch_ms.to_s
                   ^~~~~~~~

Rerun with --error-trace to show a complete error trace.
```

`epoch_ms` is removed in favour to Times `to_unix_ms`.

**Comment by duraki** [comment]
For a quick and dirty fix:

```
crystal
struct Time
  def epoch_ms
    self.to_unix_ms
  end
end
```

**Comment by duraki** [comment]
Thanks for #25 @Blacksmoke16 :)! Will try to fix and pr test soon.

**Comment by Blacksmoke16** [comment]
@duraki This is now fixed in mosquito 0.3.0.

@robacarp Can probably close this now.
